history/characters/1_azerothian.txt

```diff
@@ -17294,4 +17294,57 @@
 		employer=2 #Llane Wrynn
 	}
 	590.10.8={ death=yes }
+}
+
+# Highlord Demitrian
+5850={
+	name=Demitrian
+	dynasty=414
+	dna="ajmbbdflgbn"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=3 stewardship=4 intrigue=5 learning=6
+	add_trait=mastermind_theologian add_trait=stubborn add_trait=deceitful add_trait=erudite
+	567.3.11 = { birth=yes add_trait=creature_human }
+	583.1.1={
+		employer=2 #Llane Wrynn
+	}
+	# Joins the service of the Twilight's Hammer
+	590.11.1={
+		employer = 52000
+		religion = old_gods_worship
+		add_trait = class_shaman_5
+		effect = {
+			join_society_twilights_hammer_cult_effect = yes
+		}
+	}
+	619.10.4={ death=yes }
+}
+
+# Doctor Lapidis
+5851={
+	name=Paul
+	dynasty=415
+	dna="lejofhnijbh"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=4 stewardship=5 intrigue=5 learning=6
+	add_trait=mastermind_theologian add_trait=erudite add_trait=lunatic add_trait=shrewd
+	552.8.20 = { birth=yes add_trait=creature_human }
+	619.10.4={ death=yes }
+}
+
+# Quentin Bandor
+5852={
+	name=Quentin
+	dynasty=416
+	dna="ehckhclbeok"
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=4 stewardship=5 intrigue=4 learning=6
+	add_trait=scholarly_theologian add_trait=architect add_trait=brave add_trait=cruel
+	580.1.11 = { birth=yes add_trait=creature_human }
+	603.1.1 = {
+		employer = 62080 # Colton
```

Comment:
There's a lot of characters with this Colton as an employer but he doesn't exist.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81499467-eaebbe00-92dc-11ea-817f-4dae39ee9d19.png)

</details>